src/DwitTech.AccountService.Core/Services/SecurityService.cs

```diff
@@ -10,30 +10,23 @@ namespace DwitTech.AccountService.Core.Services
 {
     public class SecurityService : ISecurityService
     {
-        
-        public string HashString(string inputString)
-        {
-            if (inputString == null)
+         public string HashString(string inputString)
+         {
+             if (inputString == null)
```

Comment:
This should be a single 

`if (string.IsNullOrEmpty(inputString))
      return null;
`